skimage/filters/_gaussian.py

```diff
@@ -93,6 +93,9 @@ def gaussian_filter(image, sigma, output=None, mode='nearest', cval=0,
                "3D image with last dimension of length 3.")
         warnings.warn(RuntimeWarning(msg))
         multichannel = True
+    if np.any(np.asarray(sigma) < 0.0):
+        msg = "Sigma values less than zero are not valid"
```

Comment:
Right... I was expecting that to be a longer message when I started writing it. Done.